### PR TITLE
Audit logs dashboard

### PR DESCRIPTION
**Pull request by gaspergrom**
# Changes proposed ✍️

### What
copilot:summary
​
copilot:poem

### How
copilot:walkthrough

## Checklist ✅
- [ ] Label appropriately with `Feature`, `Improvement`, or `Bug`.
- [ ] Add screenshots to the PR description for relevant FE changes
- [ ] New backend functionality has been unit-tested.
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.

### CODE REVIEW

**Review by joanagmaia — COMMENTED** [review]
Code wise looks good! Will wait for the integration with backend to check out the UI

File: frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-merge.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }),
+  description: (log) => {
+    const organization = log.newState?.displayName || log.oldState?.displayName;
+    if (organization) {
+      return `${organization}<br>ID: ${log.entityId}`;
+    }
+    return '';
+  },
+  properties: (log) => {
+    const organization = log.newState?.displayName || log.oldState?.displayName;
```
Comment (joanagmaia) [inline frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-merge.ts:18]: For organizations merge and contributors merge we are now showing the entity id. It seems that log.newState.displayName doesn't exist. Maybe inside primary?
Can you check?

File: frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-edit-profile.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { LogRenderingConfig } from '@/modules/lf/config/audit-logs/log-rendering/index';
+
+function camelCaseToName(camelCase) {
+  return camelCase.replace(/([A-Z])/g, ' $1')
+    .replace(/^./, (str) => str.toUpperCase());
+}
+
+const organizationsEditProfile: LogRenderingConfig = {
+  label: 'Organization profile edited',
```
[the remark this answers is not in the document]
Comment (gaspergrom) [reply frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-edit-profile.ts:9]: fixed

File: frontend/src/modules/lf/segments/components/logs/sections/log-properties.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      <div class="w-5/12 text-sm leading-6 font-medium text-gray-500">
+        User
+      </div>
+      <div class="w-7/12 text-sm leading-24">
+        {{ props.log.user.fullName }}
+        <p class="text-2xs text-gray-500">
+          {{ props.log.user.email }}
+        </p>
+        <p class="text-2xs text-gray-500">
+          ID: {{ props.log.user.fullName }}
```
[the remark this answers is not in the document]
Comment (gaspergrom) [reply frontend/src/modules/lf/segments/components/logs/sections/log-properties.vue]: yeah had it already fixed but i guess didnt push the changes

File: frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-merge.ts (hunk 1 above)
Comment (gaspergrom) [reply frontend/src/modules/lf/config/audit-logs/log-rendering/organizations-merge.ts]: yeah the data i got from backend changes and i already aligned stuff, just didnt push the changes

**Review by joanagmaia — APPROVED** [review]

**Pull request merged**